admin_guide/out_of_resource_handling.adoc

```diff
@@ -267,6 +266,23 @@ kubeletArguments:
 
 This example is a general guideline and not recommended settings.
 
+[[out-of-resource-hard-eviction-thresholds-default]]
+===== Default Hard Eviction Thresholds
+{product-title} uses the following default configuration for `eviction-hard`.
+
+[source, yaml]
+----
+...
+kubeletArguments:
+  eviction-hard:
+  - memory.available<100Mi
+  - nodefs.available<10%
+  - nodefs.inodesFree<5%
+  - imagefs.available<15%
```

Comment:
not sure if this list is intended to be comprehensive but `imagefs.inodesFree` is missing

Reply:
@sjenning I added these details from https://github.com/sjenning/origin/blob/243298a4e8f64bb51df9df489d40fdd142f9edcb/pkg/cmd/server/kubernetes/node/node_config_test.go#L75
```
EvictionHard: "memory.available<100Mi,nodefs.available<10%,nodefs.inodesFree<5%,imagefs.available<15%",
```
It does not have any values for `imagefs.inodesFree`.

Can you suggest the value and I can probably add that (if required)

Reply:
@sjenning bump PTAL.

Reply:
Received following info from Derek 
```
imagefs.inodesFree is irrelevant until we use docker overlay by default.  
What you have doc is aligned with the default.
```
I am merging this now. If there are more changes I can update it again.